shared/README.md

```diff
@@ -42,9 +74,9 @@ After saving the file set environment using:
 . .env
```

Comment:
Environment variables listed need to be consistent with variables shown in `.tfvars` file.  See comment below.

Reply:
Its defined in the backend tfvars info. Those two are separate. Cannot be merged.